tests/figures/chart/treemap/treemap_panel_component.test.ts

```diff
@@ -0,0 +1,256 @@
+import { Model, SpreadsheetChildEnv, UID } from "../../../../src";
+import { SidePanel } from "../../../../src/components/side_panel/side_panel/side_panel";
+import { ColorGenerator } from "../../../../src/helpers";
+import { TreeMapChartDefinition } from "../../../../src/types/chart/tree_map_chart";
+import {
+  changeColorPickerWidgetColor,
+  changeRoundColorPickerColor,
+  click,
+  createTable,
+  createTreeMapChart,
+  getColorPickerWidgetColor,
+  getHTMLCheckboxValue,
+  getHTMLInputValue,
+  getRoundColorPickerValue,
+  setInputValueAndTrigger,
+  simulateClick,
+} from "../../../test_helpers";
+import {
+  openChartConfigSidePanel,
+  openChartDesignSidePanel,
+} from "../../../test_helpers/chart_helpers";
+import { mountComponentWithPortalTarget, setGrid } from "../../../test_helpers/helpers";
+
+let model: Model;
+let fixture: HTMLElement;
+let env: SpreadsheetChildEnv;
+
+function getTreeMapChartDefinition(chartId: UID): TreeMapChartDefinition {
+  return model.getters.getChartDefinition(chartId) as TreeMapChartDefinition;
+}
+
+describe("TreeMap chart side panel", () => {
+  beforeEach(async () => {
+    model = new Model();
+    createTable(model, "A1:C3");
```

Comment:
why this `createTable` ?
```suggestion
```

Reply:
I copied `waterfall_panel_component.test.ts` which had a `createTable` for whatever reason 🙈